src/storage/childtries.md

```diff
@@ -1,4 +1,109 @@
-# Cheap Inclusion Proofs: Child Tries
-*[`kitchen/modules/child-trie`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen/modules/child-trie)*
+# Cheap Membership Proofs: Child Tries
+*[`kitchen/modules/child-trie`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen/modules/child-trie)*, *[`kitchen/modules/smpl-crowdfund`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen/modules/smpl-crowdfund)*
 
-TODO: *[relevant issue (priority)](https://github.com/substrate-developer-hub/recipes/issues/35)*
\ No newline at end of file
+* [Runtime Child Storage](#storj)
+* [Crowdfund Example](#smplcrwd)
+
+A [trie](https://en.wikipedia.org/wiki/Trie) is an ordered tree structure for managing dynamic sets. For any given parent node, all descendants (children) share a common prefix associated with the parent.
+
+This construction lends itself to efficient removal of subgroups of a dataset (similar to [`double_map`](./double.md)). By associating a common prefix with related data, the dataset can be partitioned to effectively batch deletions.
+
+In addition, proofs of inclusion are relatively small in size and can be reused to verify membership of the trie in the future.
+
+Every change in the leaves percolates up to the root, thereby providing a complete, succinct history of all changes to the underlying data structure in the form of the trie root hash.
+
+## Runtime Child Storage <a name = "storj"></a>
+
+To interact with child tries, there are methods exposed in [runtime child storage](https://crates.parity.io/srml_support/storage/child/index.html). Of the methods listed in the documentation, it is worth emphasizing the method associated with batch deletion.
+
+```rust
+/// Remove all `storage_key` key/values
+pub fn kill_storage(storage_key: &[u8]) {
+	runtime_io::kill_child_storage(storage_key)
+}
+
+/// Remove value associated with `key` in trie with `storage_key`
+pub fn kill(storage_key: &[u8], key: &[u8]) {
+	runtime_io::clear_child_storage(storage_key, key);
+}
+```
+
+[`kill_storage`](https://crates.parity.io/srml_support/storage/child/fn.kill_storage.html) deletes all  `(key, value)` pairs associated with the `storage_key`. [Documentation](https://crates.parity.io/srml_support/storage/child/index.html) shows that the basic API for interacting with a given child trie follows this format:
+
+```rust
+// pseudocode
+child::do(trie_id, key, value);
+```
+
+To put an object in a child trie, the code would look something like 
+
+```rust
+fn kv_put(index: ObjectCount, who: &T::AccountId, value_type: &ValueType) {
```

Comment:
I would probably make this example independent of the rest of the code so that a user who is just reading the md file would also understand it.

Reply:
It is placing a `(key, value)` pair into the trie. What should I name the method and the variables so this is clear? I chose the names to make it easy to understand their role in the method, but I'm not the best at naming things...it's a hard problem :p

Reply:
Do you have any recommendations on variable naming to make it so that the user can understand?

Reply:
If you come up with something ping me on Riot and I can make the change. I want to merge this now as I consider it to be close enough.